components/ArtPiecePreview.js

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+
+export default function ArtPiecePreview({ artPiece }) {
+  console.log(artPiece);
+  return (
+    <>
+      <article>
+        <h3>{artPiece.name}</h3>
+        <Image
+          width={200}
+          height={200}
+          alt={`test`}
```

Comment:
maybe there is a better description  inside alt-Attribute, for example ´alt={galleryImage)´